test/stripe/core_resources/customer_test.exs

```diff
@@ -7,7 +7,8 @@ defmodule Stripe.CustomerTest do
   end
 
   test "is retrievable" do
-    assert {:ok, %Stripe.Customer{}} = Stripe.Customer.retrieve("cus_123")
+    assert {:ok, %Stripe.Customer{} = cus} = Stripe.Customer.retrieve("cus_123")
```

Comment:
`cus` - is this variable unused?

Reply:
Oops, that's a hold-over from some manual verification.  I will remove.